Sources/SpeziLLMLocal/Configuration/LLMLocalModel.swift

```diff
@@ -42,6 +42,8 @@ public enum LLMLocalModel {
     case qwen205b4bit
     /// OpenELM, 270 Million Parameters, Instruction-Tuned
     case openelm270m4bit
+    /// DeepSeek R1 Distill Qwen 1.5 Billion Parameters
+    case deepseek_r1_distill_qwen_1_5b_8bit
```

Comment:
Cool stuff, would be great to get this working as soon as #94 is merged! 🚀

Reply:
Per @LeonNissen, the 7B and 8B are also working - will add these too.